raspibolt-pulse-switch.sh

```diff
@@ -10,13 +10,14 @@ set -u
 
 # set datadir
 bitcoin_dir="/data/bitcoin"
-lnd_dir="/data/lnd"
+# Chose between LND and CLN
+ln_implemenation="CLN"
```

Comment:
```suggestion
ln_implemenation="LND"
```

I suggest LND to be the default value as it is the default client in the guide.

Reply:
I saw you resolved the conversation but parameter is still CLN. What's your view on making LND or CLN the default value?

Reply:
I would set the parameter to LND.
But while developing, i am currently only testing CLN. 
I could change it to LND everytime before i push, but i would prefer it to change it to LND at the end of the dev process.